scripts/config_core.py

```diff
@@ -103,27 +105,22 @@
     'bus_interface_type' : ['ahb_if', 'generic_bus_if', 'apb_if'],
     # Sparisty Optimizations
     'sparce_enabled' : [ 'enabled', 'disabled' ],
-    # RV32C
-    'rv32c_enabled' : [ 'enabled', 'disabled' ],
-    # base ISA Configurations
-    'base_isa': ['RV32I', 'RV32E'], 
-    
+
     # Halt Enable -- Good for testing, not for tapeout
     'infinite_loop_halts' : ['true', 'false']
   }
 
-RISC_MGMT_PARAMS = \
-  {
-    # Valid standard extensions
-    'standard_extensions' : {'name' : ['rv32m', 'rv32a', 'rv32b', 'rv32zc']},
-    # Valid nonstandard extensions
-    'nonstandard_extensions' : {'encoding' : ['R_TYPE', 'M_TYPE', 'J_TYPE', 'BR_TYPE', 'G_TYPE']}
-  }
-
-NONSTANDARD_OPCODES = [ '0001011',
-                        '0101011',
-                        '1011011',
-                        '1111011']
+# Supported ISA extensions. Order matters and should reflect the proper order
+# of extensions in an ISA string
+RISCV_ISA = {
+    "m" : False,
+    "a" : False,
+    "c" : False,
+    "b" : False,
+    "zc" : False,
```

Comment:
nitpick, this extension is actually "zicond". It's incorectly named in a few places. Should we let this go for now and fix it separately?

Reply:
Good catch, changed to `zicond` and fixed related macros. All of the other uses of `zc` can be fixed up later when there could be a conflict with a future extension.